### ISSUE TITLE
could you upload Walker2d-v2 and Humanoid-v2 PPO pretrained model?

### ISSUE THREAD

**Issue by jiameij** [ikostrikov/pytorch-a2c-ppo-acktr#80]
thanks!

**Comment by inoryy** [comment]
@jiameij with regards to your second question that's most likely due to offline `Visdom` server. See the [training](https://github.com/ikostrikov/pytorch-a2c-ppo-acktr#training) section of the readme.

**Comment by jiameij** [comment]
@inoryy thank you!

**Comment by ikostrikov** [comment]
@jiameij yes, it's because of visdom. I'm a little bit busy this and next week. I will try to upload the models in the end of the next week.

**Comment by jiameij** [comment]
@ikostrikov ok, thank you for your share!
